presto-hive/src/main/java/com/facebook/presto/hive/HiveUtil.java

```diff
@@ -211,14 +217,26 @@ private HiveUtil()
         // Tell hive the columns we would like to read, this lets hive optimize reading column oriented files
         setReadColumns(configuration, readHiveColumnIndexes);
 
+        // Only propagate serialization schema configs by default
+        Predicate<String> schemaFilter = schemaProperty -> schemaProperty.startsWith("serialization.");
+
         InputFormat<?, ?> inputFormat = getInputFormat(configuration, getInputFormatName(schema), true);
         JobConf jobConf = toJobConf(configuration);
         FileSplit fileSplit = new FileSplit(path, start, length, (String[]) null);
+        if (customSplitInfo != null && !customSplitInfo.isEmpty()) {
```

Comment:
It feels like this branch is too generic. Let's add this special extra logic only if the split is the `HoodieRealtimeFileSplit`